.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+	"useTabs": true,
+	"singleQuote": true,
+	"trailingComma": "none",
+	"printWidth": 100,
```

Comment:
Whats this?

Reply:
Code formatter. Same as revanced-website for consistency

Reply:
What is printWidth?

Reply:
Specify the line length that the printer will wrap on.